package.json

```diff
@@ -126,17 +126,18 @@
     "eslint-plugin-babel": "^5.0.0",
     "eslint-plugin-file-header": "0.0.1",
     "eslint-plugin-flowtype": "^2.36.0",
-    "eslint-plugin-jest": "^21.5.0",
+    "eslint-plugin-jest": "^21.15.1",
     "eslint-plugin-mozilla": "^0.11.0",
     "eslint-plugin-prettier": "^2.3.1",
     "eslint-plugin-react": "^7.2.1",
     "expect.js": "^0.3.1",
     "flow-bin": "0.67.1",
     "glob": "^7.0.3",
     "husky": "^0.14.2",
-    "jest": "^20.0.4",
+    "jest": "^22.4.3",
+    "jest-environment-jsdom": "^22.4.3",
     "jest-in-case": "^1.0.2",
-    "jest-junit-reporter": "^1.1.0",
```

Comment:
This reporter has gone over a year without an update and officially only supports Jest up to 18 (it runs our configs through validation so if we use any new config stuff it fails automatically.)